package/MDAnalysis/analysis/base.py

```diff
@@ -161,14 +154,15 @@ def _parallel_run(self, threads=None):
             process.join()
 
         results = []
-
         # Collects results from the queue
         while not out_queue.empty():
             results.append(out_queue.get())
 
-        for thread in sorted(results, key=itemgetter(1)):
-            self += thread[0]
```

Comment:
I didn't want to use `iadd` here because we might end up being able to add together Analysis objects to make a Manager object